### ISSUE TITLE
Standalone device?

### ISSUE THREAD

**Issue by Keckeldeckel** [home-assistant/home-assistant.github.io#29065]
### Feedback

Nice nur will there be a possibility to run the Software on a pi so you have a standalone device ?

### URL

https://www.home-assistant.io/voice_control/

### Version

2023.9.3

**Comment by dannytsang** [comment]
I'm not sure I quite get your question. Could you rephrase the question please?

**Comment by Keckeldeckel** [comment]
Is or will there be a was that i can run the Software on a different device (not the home Assistent server or any phone) so I can have like a smart speaker or something

**Comment by frenck** [comment]
Having hardware this can be used on is a goal, but we don't own a crystal ball 🪄 

The approach Home Assistant is taking, is to build usable software first. If done the other way around, we would have useless empty shells of hardware 😉 

That said, as it always goes with Home Assistant, we are building pluggable and re-usable software. That might as well be used on existing hardware or hardware you build and design yourself (e.g., using ESPHome).

../Frenck
